src/Element/TraversableElement.php

```diff
@@ -157,7 +157,11 @@ public function fillField($locator, $value)
             throw new ElementNotFoundException($this->getDriver(), 'form field', 'id|name|label|value|placeholder', $locator);
         }
 
-        $field->setValue($value);
+        if ($field->getTagName() === "select") {
```

Comment:
> BrowserKit implementation is more straightforward: https://github.com/minkphp/MinkBrowserKitDriver/blob/master/src/BrowserKitDriver.php#L414-L417

What you've specified is driver implementation. This is because the BrowserKit's `setValue` method is clever enough to understand how value should be set into the fields.